mobile/src/main/java/org/hawkular/client/android/explorer/InventoryExplorerActivity.java

```diff
@@ -65,14 +68,18 @@
     private TreeNode.BaseNodeViewHolder holder;
     private TreeNode root;
     private Callback<String> callback;
+    Toolbar toolbar;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_inventory_explorer);
+        toolbar = (Toolbar) findViewById(R.id.toolbar);
```

Comment:
@garg-anuj I have now changed the implementation of this. I am using butterknife binding 

 @BindView(R.id.toolbar)
    Toolbar toolbar;
Will update this PR soon.